android/titanium/src/java/org/appcelerator/titanium/TiC.java

```diff
@@ -1598,6 +1613,11 @@
 	/**
 	 * @module.api
 	 */
+	public static final String PROPERTY_IDENTIFIER = "identifier";
```

Comment:
ah, ok I found where this came from in the sample. In any case, it was related to the contextMenuItems stuff that we're pushing off to later.